### PR TITLE
Update cs fixer

### PR DESCRIPTION
**Pull request by alexmanno**

### CODE REVIEW

File: .php_cs.dist [hunk 1]
```diff
@@ -5,19 +5,20 @@
...
-$finder->in(array (
-  0 => 'src/',
-));
+$autoloadPathProvider = new Facile\CodingStandards\AutoloadPathProvider();
+$finder->in($autoloadPathProvider->getPaths());
 
 $config->setFinder($finder);
 
-return $config;
+return $config;
```
Comment (Jean85) [inline .php_cs.dist]: Missing new line at end of file.

**Review by Jean85 — CHANGES_REQUESTED** [review]

**Comment by Jean85** [comment]
This creates a BC, because you're dumping PHP 7 as a supported version.

**Comment by alexmanno** [comment]
> This creates a BC, because you're dumping PHP 7 as a supported version.

@Jean85 
Can this create problems?

**Comment by Jean85** [comment]
Currently, it makes the build fail: https://travis-ci.org/facile-it/sentry-psr-log/builds/439046472

**Comment by thomasvargiu** [comment]
Yes, it's a BC. I added this PR to v3.0 milestone

**Comment by thomasvargiu** [comment]
Some improvements

- `php_cs.dist` file should be added in `.gitattributes` ignored files
- we should consider to use risky rules

Probably we should do it after merging #3

**Comment by thomasvargiu** [comment]
I just saw that proposed changes are in #3. Can we add this PR into #3?

**Comment by alexmanno** [comment]
> I just saw that proposed changes are in #3. Can we add this PR into #3?

I think so!

**Comment by thomasvargiu** [comment]
Closing. See #3

**Pull request closed without merging**